about.html

```diff
@@ -39,13 +40,14 @@
   </div>
   <div class="page">
     <div class="navbar", id="topnav">
-    <ul class="navbar">
+      <ul class="navbar">
+      <a href="javascript:void(0);" class="icon" onclick="myFunction()">&#9776;</a>

```

Comment:
If we move the JS to a separate file and renamed, since it's also the same across all pages, but that's not the point of this commit. classlist.toggle will also simplify the function.